tests/integration/targets/ios_vrf_interfaces/tests/common/overridden.yaml

```diff
@@ -0,0 +1,39 @@
+---
+- ansible.builtin.debug:
+    msg: Start ios_vrf_interfaces overridden integration tests connection={{ ansible_connection}}
+
+- ansible.builtin.include_tasks: _populate_config.yaml
+
+- block:
+    - name: Override the route-policy configuration
+      cisco.ios.ios_vrf_interfaces:
+        state: overridden
+        config:
+          - name: GigabitEthernet1
+          - name: GigabitEthernet2
+          - name: GigabitEthernet3
+          - name: GigabitEthernet4
+            vrf_name: vrf_C
+
+      register: result
+
+    - name: Run show running-config | section ^interface
+      cisco.ios.ios_command:
+        commands: show running-config | section ^interface
+
+    - name: Assert that correct set of commands were generated
+      ansible.builtin.assert:
+        that:
+          - "{{ overridden['commands'] | symmetric_difference(result['commands']) |length == 0 }}"
+
+    - name: Assert that after dict is correctly generated
+      ansible.builtin.assert:
+        that:
+          - overridden['after'] == result['after']
+
+    - name: Assert that before dicts are correctly generated
+      ansible.builtin.assert:
+        that:
+          - overridden['before'] == result['before']
+  always:
```

Comment:
Lets add idempotency test

Reply:
added idempotency test for merged ,overridden and replaced